### ISSUE TITLE
Use bridges with HS under https with self signe certificates

### ISSUE THREAD

**Issue by floviolleau** [matrix-org/matrix-appservice-bridge#60]
Hi,

Thanks of providing such base for creating bridge.

I tried to use Discord / Matrix bridge (https://github.com/Half-Shot/matrix-appservice-discord) but it didn't work because I have a self-signed certificate on my HS.

When I run the discord bridge, I faced to that error on the bridge side:

```
PUT https://exemple.org/_matrix/client/r0/presence/%40_discord_107486792604712960%3Aexemple.org/status (@_discord_107486792604712960:exemple.org) HTTP null Error: {"code":"DEPTH_ZERO_SELF_SIGNED_CERT"}
```

Any clue on how to by pass this check ?
Kind regards

**Comment by V02460** [comment]
This can be closed after Matrix 1.0 I think.

**Comment by floviolleau** [comment]
Yes
